java/dagger/example/android/espresso/stubbed/StubNetworkingModule.java

```diff
@@ -0,0 +1,18 @@
+package dagger.example.android.espresso.stubbed;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.example.android.espresso.main.ApiClient;
+
+@Module
+public class StubNetworkingModule {
```

Comment:
True, but mostly overriding is completely fine while redeclaring requires either duplicating other `@Provides` methods or creating modules just for one dependency. 

Obviously composition is preferred over inheritance, but there is cost to each decision. 

Anyway, this seems covered pretty well in docs now so let's stick to "recommended" practice.